RELEASE.rst

```diff
@@ -0,0 +1,28 @@
+Release
+=======
+
+This project is released on PyPI as `coveralls`_.
+
+To cut a new release, ensure the latest master passes all tests. Then, create a release commit:
+
+1. Update the :code:`CHANGELOG.rst` with the new version.
+2. Bump the version number in :code:`setup.py`.
+3. Tag that commit with the version number (:code:`git tag x.y.z`).
```

Comment:
It is a very common practice to store the `__version__` on the init of the main module, otherwise users cannot introspect this info

https://github.com/coveralls-clients/coveralls-python/blob/master/coveralls/__init__.py#L5

so we could do something like that https://github.com/spyder-ide/qtawesome/blob/master/setup.py#L13

Or we could use versioneer